nenequest/headers/game.hpp

```diff
@@ -63,7 +53,7 @@ class Game : public Screen {
     std::vector<BreakableObject *> breakable_objects;
     BridgePit *bridge;
 
-    static constexpr float SCROLL_SPEED = -0.1;
+    static constexpr float SCROLL_SPEED = -0.1f;
```

Comment:
I don't get the purpose of the `.f` here and why it doesn't appear below.

Reply:
`f` is a suffix to designate a number as a `float`. Without it, the number is interpreted as a `double` by the compiler (which is then converted to a `float` in this line).

As for why it is not on other numbers is probably because something else caught my attention when browsing the code and forgot about it.